cicd/bootstrap.sh

```diff
@@ -79,7 +79,24 @@ add_cicd_bin_to_path() {
   if ! command -v oc_wrapper; then export PATH=$PATH:${CICD_ROOT}/bin; fi
 }
 
+check_available_server() {
+  curl -s $OC_LOGIN_SERVER > /dev/null
```

Comment:
I think you're going to want to return the exit code of this command.


```suggestion
  echo "Checking connectivity to ephemeral cluster ..."
  (curl -s $OC_LOGIN_SERVER > /dev/null)
  RET_CODE=$?
  if [ $RET_CODE -ge 1 ]; then echo "Connectivity check failed"; fi
  return $RET_CODE
```

Reply:
the last command's return code executed within a function should be the function's return value, without an explicit return, AFAIK :thinking: